trin-core/tests/overlay.rs

```diff
@@ -35,12 +35,12 @@ async fn init_overlay(
     .unwrap();
     let db = Arc::new(RwLock::new(PortalStorage::new(storage_config).unwrap()));
     let overlay_config = OverlayConfig::default();
-    let utp_listener = Arc::new(RwLockT::new(UtpListener::new(Arc::clone(&discovery))));
+    let (utp_listener_tx, _) = unbounded_channel::<UtpListenerRequest>();
```

Comment:
This is just odd enough that it might be worth a little comment, like:
```suggestion
    // Ignore all UTP events
    let (utp_listener_tx, _) = unbounded_channel::<UtpListenerRequest>();
```